meta/CXXDiagrams.m

```diff
@@ -1083,8 +1083,60 @@
 CreateVertices[
    vertices:{{__}...},
    OptionsPattern[{MaximumVerticesLimit -> 500}]] :=
-Module[{cxxVertices, vertexPartition},
-   cxxVertices = CreateVertex /@ DeleteDuplicates[vertices];
+Module[{cxxVertices, vertexPartition,
+        contextsToDistribute = {"SARAH`", "Susyno`LieGroups`", "FlexibleSUSY`", "CConversion`", "Himalaya`"}},
+
+   (* without this CForm includes context in name of symbols
+      such that we get for example SARAH_g1 instead of g1 in
+      generated C++ code *)
+   ParallelEvaluate[
+      (BeginPackage[#];EndPackage[];
+       (* prevent shdw warning with Susyno`LieGroups`M: *)
+       Off[Remove::remal];
+       Remove[Susyno`LieGroups`M];
+       On[Remove::remal];
+      )& /@ contextsToDistribute,
+      DistributedContexts->Automatic
+   ];
+   (* without this CForm converts complex numbers using
+      Complex wrapper *)
+   ParallelEvaluate[
+      Unprotect[Complex];
```

Comment:
This code exists inside `CConversion.m` already, so I think, that it should not be just copy-pasted here from there. I have found a safer way to do similar thing in #269.